rpcd/playbooks/roles/rpc_support/tasks/main.yml

```diff
@@ -95,3 +95,4 @@
 
 - include: configure_unattended_upgrades.yml
 
+- include: create_default_flavors.yml
```

Comment:
Hmm...That's strange to me since this is the expression used throughout the file. i.e. https://github.com/rcbops/rpc-openstack/blob/master/rpcd/playbooks/roles/rpc_support/tasks/main.yml#L64